astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -90,12 +90,13 @@ def _make_transform_graph_docs():
     frames) is set by the type of transformation; the legend box defines the
     mapping from transform class name to color.
 
+    .. container:: gvcustom
```

Comment:
Add a comment (move the one you added to the changelog) to just above this line

Reply:
OKay, Will do it.